build-support/bin/release.sh

```diff
@@ -7,7 +7,7 @@ set -e
 ROOT=$(cd $(dirname "${BASH_SOURCE[0]}") && cd "$(git rev-parse --show-toplevel)" && pwd)
 source ${ROOT}/build-support/common.sh
 
-PY=$(which python2.7)
+PY=$(which python2.7 || exit 0)
```

Comment:
I'd maybe write this as:
```
PY=$(which python2.7 || die "You must have python2.7...")
export PY
```

because exiting 0 when what you want to do is fail seems a little weird :)

Reply:
AFAIK, this exits the subshell, not the whole script. This is the actual fix for the issue here... the changes in `pants_run_integration` are just noise.

Reply:
Er, same thing applies there I think. We'll die on the next line with the message.